stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/common/extensions/internal/Context.kt

```diff
@@ -63,18 +63,22 @@ internal val Context.streamThemeInflater: LayoutInflater
 
 internal fun Context.createStreamThemeWrapper(uiMode: UiMode = ChatUI.uiMode): Context {
     val typedValue = TypedValue()
-    val configuredContext = configureUiMode(uiMode)
 
     return when {
-        configuredContext.theme.resolveAttribute(R.attr.streamUiValidTheme, typedValue, true) -> configuredContext
-        configuredContext.theme.resolveAttribute(R.attr.streamUiTheme, typedValue, true) ->
-            ContextThemeWrapper(configuredContext, typedValue.resourceId)
-        else -> ContextThemeWrapper(configuredContext, R.style.StreamUiTheme)
+        this.theme.resolveAttribute(R.attr.streamUiValidTheme, typedValue, true) ->
+            ContextThemeWrapper(this, R.style.StreamUiEmptyTheme)
```

Comment:
If it is a valid theme, we shouldn't wrap the context and apply a new theme on it

Reply:
The problem is that we need `ContextThemeWrapper` to use `applyOverrideConfiguration`. At this moment I didn't find a other way to do this.

Reply:
What about to use the current theme?
Can't you do something like:
```
ContextThemeWrapper(this, this.theme)
```

Reply:
This requires API 23 😢

Reply:
We need to find another solution, if we apply an Empty Theme, we will override the valid theme and nothing will be configured with the theme

Reply:
From what I read, the new theme doesn't override fields that are not the same. I did some tests and changing the base theme fields have effect in the sample app... but we would probably lose runtime changes in the theme. 

The other solution I see would be to pass a flag to our Style classes and parse the right color there. We could have literal colors (literal_white, literal_black, literal_blue...) and we choose between then. The draw back in that it will need more code and it is a bit harder to maintain... but on the good side it removes the need for the `StreamUiEmptyTheme` (which is super odd).

Reply:
@JcMinarro What do you think? 👆

Reply:
So.... Whatever field doesn't exis on the new theme is asked to the "base theme" of the context?
On that case it should be fine. 

The second option you propose we should avoid, we shouldn't add any logic in our Style Code to decide which color to use, it should be handled by the system